tools/check_spelling.sh

```diff
@@ -36,13 +39,29 @@ cd "$ROOTDIR"
 BIN_FILENAME="misspell_${VERSION}_${OS}_64bit.tar.gz"
 # Install tools we need
 if [[ ! -e "/tmp/misspell" ]]; then
```

Comment:
One more thing -- can you comment why you picked 'misspell' here rather than [aspell](http://aspell.net/) or other engines that might be installable with apt-get or brew?

Reply:
OK, cool. Can you add that as a comment here in the script?

One other thought: there are instructions on setting up a dev environment in https://github.com/envoyproxy/envoy/blob/master/DEVELOPER.md .  I wonder if it makes sense to instruct devs to manually install misspell when setting up their env.  The answer may be 'no', but e should -- as a follow-up, see what senior maintainers prefer.